### PR TITLE
Updated docs for TLS module

### PR DESCRIPTION
**Pull request by adriansr**
Updated packetbeat docs regarding the TLS module.

I didn't see anything else to update but I might have skipped something. There is a `tls-dashboard.png` image but I don't see where to put it.

### CODE REVIEW

File: packetbeat/docs/packetbeat-options.asciidoc [hunk 1]
```diff
@@ -1090,6 +1092,48 @@ at the end of the document.
...
+==== Configuration options
+
+The `send_certificates` and `include_raw_certificates` settings are useful
+for limiting the amount of data Packetbeat indexes in its certificate fields.
+
+Also see <<common-protocol-options>>.
+
+===== `send_certificates`
+
+If the certificates presented by the client and server must be included in the
```
Comment (dedemorton) [inline packetbeat/docs/packetbeat-options.asciidoc]: You have a sentence fragment here. Did something get deleted by mistake? Did you mean to say, "Specify this setting if the certificates....?

File: packetbeat/docs/packetbeat-options.asciidoc [hunk 2]
```diff
@@ -1090,6 +1092,48 @@ at the end of the document.
...
+event. The server's certificate is indexed in the `server_certificate` field and
+its certification chain in the `server_certificate_chain`. For the client,
```
Comment (dedemorton) [inline packetbeat/docs/packetbeat-options.asciidoc]: Change to: "the `server_certificate_chain` field.

File: packetbeat/docs/packetbeat-options.asciidoc [hunk 3]
```diff
@@ -1090,6 +1092,48 @@ at the end of the document.
...
+------------------------------------------------------------------------------
+packetbeat.protocols:
+- type: tls
+  send_certificates: true
+  include_raw_certificates: true
+------------------------------------------------------------------------------
+
+==== Configuration options
+
+The `send_certificates` and `include_raw_certificates` settings are useful
```
Comment (dedemorton) [inline packetbeat/docs/packetbeat-options.asciidoc]: Might help to add another sentence or two here because it's not immediately clear to me how these fields limit the amount of data indexed. Maybe it would help to explain the default behavior when users don't specify these fields. If you think this detail is too much to include in the description, maybe consider including a summary in the intro to this topic.

File: packetbeat/docs/packetbeat-options.asciidoc [hunk 4]
```diff
@@ -1090,6 +1092,48 @@ at the end of the document.
...
+If the certificates presented by the client and server must be included in the
+event. The server's certificate is indexed in the `server_certificate` field and
+its certification chain in the `server_certificate_chain`. For the client,
+`client_certificate` and `client_certificate_chain` are used.
+
+The default is `true`.
+
+===== `include_raw_certificates`
+
+In addition to a fields representation of each certificate, include also the
```
Comment (dedemorton) [inline packetbeat/docs/packetbeat-options.asciidoc]: Maybe switch this around a bit because it's a little hard to parse at first:

When `send_certificates` is `true`, you can set `include_raw_certificates` to include the raw certificate encoded in PEM format as a `raw` field under each certificate field.

If `send_certificates` is `false`, this setting is ignored. The default is `false`.

File: packetbeat/docs/packetbeat-options.asciidoc [hunk 5]
```diff
@@ -1090,6 +1092,48 @@ at the end of the document.
 Note that limiting documents in this way means that they are no longer correctly
 formatted JSON objects.
 
+[[configuration-tls]]
+=== Capture TLS traffic
+
+++++
+<titleabbrev>TLS</titleabbrev>
+++++
+
```
Comment (andrewkroh) [inline packetbeat/docs/packetbeat-options.asciidoc]: I think we need an overview of the feature that explains what it does and how it does. Some points to possibly cover:

- It captures TLS handshake data.
- Make it clear that this does not decrypt any traffic.
- It provides information about the TLS versions, ciphers, and certificates being used. It measures the time it takes for the handshake to complete.
- It computes a JA3 fingerprint of the client's TLS parameters. The fingerprint can be used to identify an application associated with the traffic. The fingerprint is independent of network addresses and certificates.

File: packetbeat/docs/packetbeat-options.asciidoc (hunk 1 above)
Comment (adriansr) [reply packetbeat/docs/packetbeat-options.asciidoc]: No, I'm just a lousy writer.

File: packetbeat/docs/packetbeat-options.asciidoc (hunk 5 above)
Comment (adriansr) [reply packetbeat/docs/packetbeat-options.asciidoc]: I will leave the JA3 and handshake time for another patch so this one can be backported to 6.1.

**Comment by adriansr** [comment]
@dedemorton I condensed the event a little bit, hope that is enough.

Please feel free to comment on all those nit picks or better, just rewrite yourself anything you don't like.
